From 6acc08baa341fcd8f4d255c2d10f0be62f565b8d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 13 Apr 2015 17:02:13 -0700
Subject: [PATCH] Unset need_charset_alias when building for musl

localcharset uses ac_cv_gnu_library_2_1 from glibc21.m4
which actually shoudl be fixed in gnulib and then all downstream
projects will get it eventually. For now we apply the fix to
coreutils

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 lib/gnulib.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/gnulib.mk b/lib/gnulib.mk
index 3346e0d..65eeca3 100644
--- a/lib/gnulib.mk
+++ b/lib/gnulib.mk
@@ -824,7 +824,7 @@ install-exec-localcharset: all-local
 	  case '$(host_os)' in \
 	    darwin[56]*) \
 	      need_charset_alias=true ;; \
-	    darwin* | cygwin* | mingw* | pw32* | cegcc*) \
+	    darwin* | cygwin* | mingw* | pw32* | cegcc* | linux-musl*) \
 	      need_charset_alias=false ;; \
 	    *) \
 	      need_charset_alias=true ;; \
